docs/dev/api.rst

```diff
@@ -25,60 +40,178 @@ The Library Class
 
     .. automethod:: get_album
 
-    .. automethod:: add
-
-    .. automethod:: add_album
+    Any modifications must go through a :class:`Transaction` which you get can
+    using this method:
 
     .. automethod:: transaction
 
-Transactions
-''''''''''''
-
-The :class:`Library` class provides the basic methods necessary to access and
-manipulate its contents. To perform more complicated operations atomically, or
-to interact directly with the underlying SQLite database, you must use a
-*transaction*. For example::
-
-    lib = Library()
-    with lib.transaction() as tx:
-        items = lib.items(query)
-        lib.add_album(list(items))
-
-.. currentmodule:: beets.dbcore.db
+.. _SQLite: http://sqlite.org/
+.. _ORM: http://en.wikipedia.org/wiki/Object-relational_mapping
 
-.. autoclass:: Transaction
-    :members:
 
 Model Classes
 -------------
 
 The two model entities in beets libraries, :class:`Item` and :class:`Album`,
-share a base class, :class:`Model`, that provides common functionality and
+share a base class, :class:`LibModel`, that provides common functionality and
 ORM-like abstraction.
 
-The fields model classes can be accessed using attributes (dots, as in
-``item.artist``) or items (brackets, as in ``item['artist']``). The
-:class:`Model` base class provides some methods that resemble `dict`
-objects.
```

Comment:
I actually think this is nice to mention here—and perhaps not below, in the two places where we say something similar about the `Item` and `Album` classes. They really do work the same way, and it would be nice to emphasize that the main way to interact with beets data is by accessing these attributes.

Reply:
Agreed. I've moved it back to the top and edited the wording slightly to emphasise that the behaviour is generic to models.